packages/dd-trace/src/config.js

```diff
@@ -972,6 +968,25 @@ class Config {
     }
   }
 
+  _setSamplingRule (obj, name, value) {
+    if (value === null || value === undefined) {
```

Comment:
FYI this is equivalent to:
```suggestion
    if (value == null) {
```
up to you to decide which one you prefere